oxidation/crates/platform_native/tests/test_transport.rs

```diff
@@ -0,0 +1,31 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+use async_std::net::TcpListener;
+use async_std::task;
+use libparsec_platform_native::Transport;
+use rstest::rstest;
+
+#[rstest]
+fn test_transport() {
+    smol::block_on(async {
+        let addr = "127.0.0.1:8080";
```

Comment:
Should not pick a fixed port, this typically prevent concurrent tests

Instead should use `TcpListener::bind("127.0.0.1:0")` to have the OS assign a port for us

Reply:
Done :+1: